mantidproject/mantidimaging
# Automatically applies minus log before reconstruction

Also removes the `minus_log` operation as it is no longer necessary.

To test:
- Load a dataset
- Go right into reconstruction window
- Run correlate to get some COR
- Reconstructed image on the right should look OK and the values in a range of around 0.000-0.0100 (depends on sample)
- Run a full recon and see that the output looks the same as the image in the preview

Fixes #550
